lib/src/yaml_edit/yaml_edit.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// YAML parsing is supported by `package:yaml`, and each time a change is
+/// made, the resulting YAML AST is compared against our expected output
+/// with deep equality to ensure that the output conforms to our expectations.
+///
+/// **Example**
+/// ```dart
+/// import 'package:yaml_edit/yaml_edit.dart';
+///
+/// void main() {
+///  final yamlEditor = YamlEditor('{YAML: YAML}');
+///  yamlEditor.update(['YAML'], "YAML Ain't Markup Language");
+///  print(yamlEditor);
+///  // Expected Output:
+///  // {YAML: YAML Ain't Markup Language}
+/// }
+/// ```
+///
+/// [1]: https://yaml.org/
+library yaml_edit;
+
+export 'src/editor.dart';
+export 'src/equality.dart' show deepEquals;
```

Comment:
What do we export this? What use does it have end-users?

Reply:
hm yeah you're right - I exported it thinking that there might be causes where users need to compare two YamlNodes for equality of their values but I guess that might not be as important. Will remove!